multimedia/mlt/Portfile

```diff
@@ -158,8 +158,12 @@ variant gpl3 description {enable GPLv3 components} {
 # the opencv dependency must match our choice of Qt version, which is
 # another reason why opencv support is provided through a variant.
 variant opencv description {enable OpenCV support} {
-    depends_lib-append      port:opencv
+    set opencv_ver          3
+    depends_lib-append      port:opencv${opencv_ver}
+    configure.env-append    PKG_CONFIG_PATH=${prefix}/lib/opencv${opencv_ver}/pkgconfig
     configure.args-delete   --disable-opencv
+    configure.args-append   --enable-opencv
```

Comment:
Those last two lines can be condensed to

```
configure.args-replace  --disable-opencv   --enable-opencv
```

Reply:
Done; change pushed.

Reply:
p.s. Thanks for catching this Chris!